### DISCUSSION TITLE
any tips for getting the "borg" out ???

### DISCUSSION THREAD

**Discussion by utunga** [coqui-ai/TTS#679] [General Q&A]
Thought I'd give an example of my results and ask if anyone can help with the de-borgification of our model?

This was training with 6.5 hrs or so of studio quality sentence/audio pairs from a single speaker. Using `0.1.2` coqui_tts (with obvious mods to add our dataset etc). Tried a few different things but this is the output of the tacotron2_ddc model. 

Still not especially happy with this result especially that dang borg. How do we get the borg out ;_) ? Seriously though, just thought I'd give the status update and put it out there in case:
- anyone has suggestions..  ? (anything welcomed!)
- anyone has comments on whether 6.5 hrs is enough audio to train a single speaker model?

Here's a sample of what it sounds like at 1000 epochs (yes we're working on a new tts model for te reo Māori) 

https://user-images.githubusercontent.com/166867/126934208-c62f71c4-deee-4b00-bff3-3157714fae55.mov

This has been converted from wav to mov to embed nicely in this forum, but sounds the same as the raw [wav output](https://github.com/TeHikuMedia/coqui_tts/blob/papareo_branch/tts_train_epoch_1000.wav)

As you can see there's got quite good alignment but the loss functions seemed to kinda 'bottom out' (or actually be going up) well before the full 1000 epoch train. 

![TensorBoard - 54 206 123 58_alignment](https://user-images.githubusercontent.com/166867/126934635-f5d275cb-25e0-4102-a1b7-650ee0f0c787.png)

![TensorBoard - 54 206 123 58](https://user-images.githubusercontent.com/166867/126934649-bead678e-b5fe-4b4c-b770-e5b63bd8e36e.png)

You can see the core configs i used [here](https://github.com/TeHikuMedia/coqui_tts/blob/papareo_branch/docs/papareo_t2_ddc_config.json) 

Any suggestions very much welcomed in advance. Otherwise will keep plugging away.

Thanks!

**Comment by erogol** [comment]
What is borg?

**Reply by thorstenMueller** [reply]
I think he meant the alien species (_machine and biological mix_) from Star Trek which speaks mechanically ;-).
https://en.wikipedia.org/wiki/Borg

**Reply by utunga** [reply]
Sorry @erogol an attempt at a joke... i just mean that it sounds like a robot voice - if you listen to the sample. Something a bit 'borg' like in the way it comes out. Is it audio out of phase or something? Still lacking proper alignment? I dunno.

Can I fix this by switching up the vocoder? Losing tacotron2 and going to glowtts.. any recommendations for a single voice model that might be happy with 'only' 6.5 hours of training data?

**Reply by erogol** [reply]
Sorry, I couldn't "borged" your joke out. My fault :) 

I guess the example above is without a vocoder. Try with a vocoder. 
You can start with using one of the universal vocoders we released. If nothing works, train a new one. 

It looks to me that the "borgy" voice is about not using a vocoder.
